log/log.go

```diff
@@ -39,10 +41,57 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setFormatterSylog(appname, local string) error {
+	if appname != "" && local != "" {
+		fmter, err := newSyslogger(appname, local, origLogger.Formatter)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "error creating syslog formatter: %v\n", err)
+			origLogger.Errorf("can't connect logger to syslogs: %v", err)
+		} else {
+			origLogger.Formatter = fmter
+		}
+	}
+	return nil
```

Comment:
Try to reduce indentation in go.

```go
if appname == "" || local == "" {
  return nil
}

fmter, err := newSyslogger(appname, local, origLogger.Formatter)
if err != nil {
  fmt.Fprintf(os.Stderr, "error creating syslog formatter: %v\n", err)
  origLogger.Errorf("can't connect logger to syslogs: %v", err)
  return nil
}

origLogger.Formatter = fmter
return nil
```

I don't really understand, why your method signature includes a `error` return value. You never check for the return value and always return `nil` anyway.